restapi/src/test/java/io/stargate/web/docsapi/dao/DocumentDBTest.java

```diff
@@ -83,14 +83,57 @@ public void getForbiddenCharactersMessage() {
 
   @Test
   public void containsIllegalChars() {
-    assertThat(DocumentDB.containsIllegalChars("[")).isTrue();
-    assertThat(DocumentDB.containsIllegalChars("]")).isTrue();
-    assertThat(DocumentDB.containsIllegalChars(",")).isTrue();
-    assertThat(DocumentDB.containsIllegalChars(".")).isTrue();
-    assertThat(DocumentDB.containsIllegalChars("\'")).isTrue();
-    assertThat(DocumentDB.containsIllegalChars("*")).isTrue();
-    assertThat(DocumentDB.containsIllegalChars("a[b")).isTrue();
-    assertThat(DocumentDB.containsIllegalChars("\"")).isFalse();
+    // first some positive matches
```

Comment:
These might be a good candidate for a parameterized test.

```java
  @ParameterizedTest
  @ValueSource(strings = {"[", "]", ... })
  public void containsIllegalChars(String str) {
     // ...
```

Reply:
Good point, I can change that.

Reply:
Changed, split into couple of separate test methods to keep more manageable.